index.js

```diff
@@ -8,7 +8,7 @@
 
 'use strict'
 
-const base64 = require('base64-js')
+const hextreme = require('hextreme')
```

Comment:
A new dependency as non-battle-tested as a newly published package - unfortunately - isn't tenable for `Buffer`.

The `TextDecoder` option presented previously was an easy option as the complexity of review remained in band.